docs/modules/ROOT/pages/sentinel.adoc

```diff
@@ -515,6 +515,21 @@ NOTE: It can take several minutes for a new custom metric to show up in the Data
 
 * *Metric Prefix* will precede all metric names.  For instance, with a prefix of `defender.`, sentinels will send a metric called `defender.sentinel`.
 
+=== PagerDuty Configuration
+
+Please see https://support.pagerduty.com/docs/services-and-integrations[PagerDuty integration documentation] to configure an PagerDuty API v2 integration that can create change and alert events.
+
+* *Event Type* Event type for pager duty categorization (alert or change)
+* *Routing Key* Integration Key for an integration on a service or on a global ruleset (32 characters)
+* *Event Action* The action type of event (trigger, acknowledge or resolve)
+* *Dedup Key* Deduplication key for correlating triggers and resolves (255 max characters)
+* *Severity* The perceived severity of the status the event is describing with respect to the affected system  (critical, error, warning or info)
```

Comment:
2 whitespaces between `system` and the brackets

Reply:
Hope you don't mind, just went ahead and removed it.

Reply:
Thank you very much! Did not see it on vs code will be more careful.
Thanks for the fix!